synapse/handlers/federation_event.py

```diff
@@ -939,20 +939,26 @@ async def _get_state_ids_after_missing_prev_event(
         )
 
         logger.debug(
-            "state_ids returned %i state events, %i auth events",
+            "_get_state_ids_after_missing_prev_event(event_id=%s): state_ids returned %i state events, %i auth events",
```

Comment:
Just giving all of these a namespaced/scoped context to easily compartmentalize them and follow down the rabbit hole.